### PR TITLE
Update and improve readme

### PR DESCRIPTION
**Pull request by MrGeneration**
With switching branches, the readme needed an update. Also user and admin docs both have different information on the same portions which also leads in inconsistencies. This commit should solve these.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,47 @@
...
 - http://www.sphinx-doc.org/en/stable/rest.html
 - http://docs.readthedocs.io/en/latest/_themes/sphinx_rtd_theme/demo_docs/source/demo.html
 
-Thanks! ❤️ ❤️ ❤️
+### Versioning
+
+This documentation provides versions:
 
-  Zammad Team
+- ``pre-release`` can contain develop, not yet releases functions and changes
```
Comment (mgruner) [inline README.md:35]: not yet `released`

**Review by mgruner — APPROVED** [review]
Looks great, one minor typo.

File: README.md (hunk 1 above)
Comment (MrGeneration) [reply README.md:35]: good call. Will fix that in both versions.

File: README.md [hunk 2]
```diff
@@ -2,36 +2,47 @@
 
 # Zammad admin documentation
```
Comment (mgruner) [inline README.md:3]: Hm, we might want to capitalize this too.
But OTOH, if you don't use capitalization style consistently yet in the docs, we can keep it like it is for now.

Comment (mgruner) [reply README.md]: "Title case" that is. https://capitalizemytitle.com/

Comment (MrGeneration) [reply README.md]: No it's a good point and addressed already. :)

**Pull request merged**